src/main/java/gov/cms/mat/cql_elm_translation/cql_translator/TranslationResource.java

```diff
@@ -112,19 +124,28 @@ public CqlTranslator buildTranslator(
         }
       }
 
-      CqlTranslatorOptions.Options[] options =
-          optionsList.toArray(new CqlTranslatorOptions.Options[0]);
+      CqlCompilerOptions.Options[] options = optionsList.toArray(new CqlCompilerOptions.Options[0]);
 
       libraryManager.getLibrarySourceLoader().registerProvider(new MadieLibrarySourceProvider());
-
-      return CqlTranslator.fromStream(
-          cqlStream,
-          modelManager,
-          libraryManager,
-          ucumService,
-          CqlCompilerException.ErrorSeverity.Error,
-          signatureLevel,
-          options);
+      // this was the old code for version 2.11.0 of cqframework.. the constructor changed
+      // drastically, so want to save this until we get past any regressions problems
+      /*public static CqlTranslator fromStream(NamespaceInfo namespaceInfo, VersionedIdentifier sourceInfo, InputStream cqlStream,
+      LibraryManager libraryManager) throws IOException {*/
+      //      return CqlTranslator.fromStream(
+      //          cqlStream,
+      //          modelManager,
+      //          libraryManager,
+      //          ucumService,
+      //          CqlCompilerException.ErrorSeverity.Error,
+      //          signatureLevel,
+      //          options);
+
+      NamespaceInfo nsInfo = null;
+
+      libraryManager.setUcumService(ucumService);
+      libraryManager.getCqlCompilerOptions().setOptions(options);
+
+      return CqlTranslator.fromStream(nsInfo, sourceInfo, cqlStream, libraryManager);
```

Comment:
Regarding setting `SignatureLevel` and `ErrorSeverity`, looks like `CqlCompilerOptions` has a constructor that accepts both along with other options. I was wondering if we can use them, Do we need them ? idk whats the importance of these configs. 

1. We can ask the Bryn's team and gather some knowledge on these configs.
2. We can push it as is, wait and see if there will be any Helpdesk tickets. Since we don't even how to test these configs, I am ok with going ahead with what you have.

   ```
 /**
     * Constructor with defined ErrorSeverity, SignatureLevel, and arbitrary number of options.
     *
     * @param errorLevel
     * @param signatureLevel
     * @param options
     */
    public CqlCompilerOptions(CqlCompilerException.ErrorSeverity errorLevel, LibraryBuilder.SignatureLevel signatureLevel, Options... options) {
        this.setOptions(options);
        this.errorLevel = errorLevel;
        this.signatureLevel = signatureLevel;
    }
```